discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use tokio::sync::RwLock;
+use xayn_ai::ranker::KeyPhrase;
+
+use crate::{
+    document::Document,
+    engine::{EndpointConfig, GenericError, Market},
+    ranker::Ranker,
+    stack::Id,
+};
+
+use super::Ops;
+
+/// Stack operations customized for breaking news items.
+// NOTE mock implementation for now
+struct BreakingNews {
+    markets: Option<Arc<RwLock<Vec<Market>>>>,
+}
+
+#[async_trait]
+impl Ops for BreakingNews {
+    fn id(&self) -> Id {
+        todo!();
+    }
+
+    fn configure(&mut self, config: EndpointConfig) {
+        let _prev_mkts = self.markets.replace(config.markets);
```

Comment:
since the replaced value is not `must_use`, we could just omit it. similarly for the `PersonalizedNews` impl.
```suggestion
        self.markets.replace(config.markets);
```